HTML+CSS+JS/Motion Background/Readme.md

```diff
@@ -0,0 +1,8 @@
+# issue
+done with the motion background (html + css )
+# used 
+you can use any component as per your wish even from google just copy the path and paste it in place the element currently used.
+# demo
+
```

Comment:
Rename the file to `README.md` otherwise GitHub cannot detect it.

Also use this template to make the README file for you component- https://github.com/ALPHAVIO/Mirage-UI/blob/master/README_TEMPLATE.md

Take help from this README- https://github.com/ALPHAVIO/Mirage-UI/tree/master/HTML%2BCSS%2BJS/Buttons/BlueBackgroundOutButtonAnimation